app/Console/Commands/GenerateCouponMoney.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Models\Coupon;
+use Carbon\Carbon;
+use Illuminate\Support\Str;
+
+class GenerateCouponMoney extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'coupon:money {discount=10} {min=20} {day=3}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generation coupon money';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $coupon = [
+            'discount' => $this->argument('discount'),
```

Comment:
ở dưới này em mới xét để gán giá trị, như:
`$this->argument('discount') ? $this->argument('discount') : DISCOUNT;`
Tương tự cho các biến kia